chapters/packages.tex

```diff
@@ -322,7 +322,7 @@ \section{External Resources}\label{external-resources}
 The references \filename{modelica://Modelica.Mechanics/C.jpg} and \filename{modelica://Modelica/C.jpg} must also refer to two distinct resources.
 \end{example}
 
-\section{Multilingual Descriptions}\label{multilingual-descriptions}
+\section{Multilingual Descriptions}\label{multilingual-descriptions}\index{translation}
```

Comment:
The main meaning of _translation_ is a different one in our document, so this index entry so be differentiated somehow, for example:
```suggestion
\section{Multilingual Descriptions}\label{multilingual-descriptions}\index{translation!multilingual libraries}
```

At the same time, we should also qualify the entry for the main meaning:
```
\begin{definition}[Translation]\index{translation!of simulation model}
Translation is the process of preparing a Modelica simulation model for simulation, starting with flattening but not including the simulation itself.
\end{definition}
```

By the way, isn't the section title a bit narrow considering that many more things than just description strings can be translated?  Wouldn't _Multilingual Libraries_ be a more descriptive section title?

Reply:
I agree that we might reconsider it for the future, but "Multilingual Libraries" doesn't seem ideal either.